roles/reproducer/tasks/validations.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Ensure we pass needed parameter
+  ansible.builtin.assert:
+    that:
+      - cifmw_use_libvirt is defined
+      - cifmw_use_libvirt | bool
+    msg: >-
+      Please ensure you pass "cifmw_use_libvirt: true" to the
+      deployment!
+    quiet: true
+
+- name: Verify ansible_host is defined in inventory
+  when: cifmw_target_host != 'localhost'
+  delegate_to: localhost
```

Comment:
in fact, we should loop on all the known inventory: we need the `ansible_host` for remote nodes as well.